arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentTableViewController.swift

```diff
@@ -188,7 +188,12 @@ class ContentTableViewController: UITableViewController, CustomSearchHeaderViewD
                 strongSelf.downloadProgressView.dismiss()
                 
                 if let error = error {
-                    SVProgressHUD.showError(withStatus: "Failed to download raster resource :: \(error.localizedDescription)")
+                    if let indexPath = strongSelf.tableView.indexPathForSelectedRow {
+                        strongSelf.tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
Selection is not getting cleared on cancelling the ODR download. Debugger is not hitting this breakpoint on cancelling the process